imagefeedview/src/test/java/com/oliveroneill/imagefeedview/ImageFeedViewTest.kt

```diff
@@ -20,14 +18,14 @@ class ImageFeedViewTest {
 
     @Before
     fun setup() {
-        mockGrid = mock()
-        mockPager = mock()
-        feed = ImageFeedView(mock())
-        feed.setGridAdapter(mockGrid, mock())
-        feed.setPagerAdapter(mockPager, mock())
-        feed.initGridAdapter(mock(), 3)
-        feed.initPagerAdapter(mock())
-        feed.setSwipeRefresh(mock())
+        mockGrid = mockk(relaxed = true)
```

Comment:
relaxed mock required if you are not going to specify mock behavior explicitly, by default mocks are strict